test/integration/features/basic.feature

```diff
@@ -28,17 +30,14 @@ Checks whether CRC top-level commands behave correctly.
     And stdout should contain "Installing crc-driver-libvirt"
     And stdout should contain "Setting up Libvirt crc network"
     And stdout should contain "Starting Libvirt crc network"
-    
+
   Scenario: CRC start
-    When executing "crc start -b ~/Downloads/crc_libvirt_v4.1.0.rc0.tar.xz" succeeds
-    Then stdout should contain "Creating VM"
-    And stdout should contain "Running"
+    When executing "crc start -b ~/Downloads/crc_libvirt_4.1.0-rc.5.tar.xz" succeeds
```

Comment:
Instead of that always download it on the current crc binary directory and use `crc start -b crc_libvirt_4.1.0-rc.5.tar.xz` but again this will be differ on the host. so you might have to put a logic around it.